Doc/whatsnew/3.7.rst

```diff
@@ -441,6 +441,19 @@ can be set within the scope of a group.
 ``'^$'`` or ``(?=-)`` that matches an empty string.
 (Contributed by Serhiy Storchaka in :issue:`25054`.)
 
+ssl
+---
+
+Added support for validating server certificates containing
+internationalized domain names (IDNs). As part of this change, the
+:attr:`ssl.SSLSocket.server_hostname` and
+:attr:`ssl.SSLObject.server_hostname` attributes now store the
+expected hostname in A-label form (``"xn--pythn-mua.org"``), rather
+than the U-label form (``"pythön.org"``). The server name argument to
+:meth:`ssl.SSLContext.set_servername_callback`` is an A-label instead of
+an U-labe, too. (Contributed by Nathaniel J. Smith and Christian Heimes
```

Comment:
"labe"

Reply:
(also not sure what the deal with github rendering all this with underlines is; potential reST issue?)

Reply:
It's caused by a supernumerous backtick. I have fixed both issues.